### ISSUE TITLE
Wunsch: Hinweis im FE ausblenden

### ISSUE THREAD

**Issue by svdoldie** [diddipoeler/sportsmanagement#1253]
Es wäre schön, wenn man die Hinweise auch im Frontend ausblenden könnte.
Ich habe die Funktion nur in den Optionen für das Backend finden können

**Comment by diddipoeler** [comment]
Welche Hinweise ?

**Comment by svdoldie** [comment]
In der Ansicht Verinsinfo z.b https://www.svdoerpum.de/index.php/verein/clubinfo

**Comment by diddipoeler** [comment]
Soll da nur der Text: Hinweis weg ?
Was danach kommt ist ja der eigentliche Text.

**Comment by svdoldie** [comment]
Nein, ich meine die komplette Zeile. Wenn man die Ansicht als Tabs anzeigt,
steht der Text schon in der Überschrift der Tabs.
Der Text Hinweis kann z.B. bei Mannschaften des Vereins generell weg

**Comment by ortwin20000** [comment]
Du könntest dies auch mit Hilfe eines Template-Overrides für die einzelnen Tabs erledigen:
Z.B. für das Tab Google Maps:
Ein Override für components\com_sportsmanagement\views\globalviews\tmpl\default_googlemap.php erstellen und danach die Zeile `<div class="info-tab note-icon" title="<?php echo Text::_('COM_SPORTSMANAGEMENT_GMAP_DIRECTIONS'); ?>"><i></i></div>` auskommentieren.

**Comment by svdoldie** [comment]
Und mir ist aufgefallen, das die Hinweise jetzt auch nicht angezeigt werden, obwohl die Option eingeschaltet ist
